Change default paths
--- a/bin/run.sh
+++ b/bin/run.sh
@@ -3,12 +3,11 @@
 # Simple shell-script to run HornetQ standalone
 #------------------------------------------------
 
-export HORNETQ_HOME=..
-mkdir -p ../logs
+. /etc/default/hornetq
 
 # By default, the server is started in the non-clustered standalone configuration
 
-if [ a"$1" = a ]; then CONFIG_DIR=$HORNETQ_HOME/config/stand-alone/non-clustered; else CONFIG_DIR="$1"; fi
+if [ a"$1" = a ]; then CONFIG_DIR=$HORNETQ_HOME/config/$HORNETQ_TYPE; else CONFIG_DIR="$1"; fi
 if [ a"$2" = a ]; then FILENAME=hornetq-beans.xml; else FILENAME="$2"; fi
 
 if [ ! -d $CONFIG_DIR ]; then
@@ -22,14 +21,14 @@
 # Use the following line to run with different ports
 #export CLUSTER_PROPS="-Djnp.port=1099 -Djnp.rmiPort=1098 -Djnp.host=localhost -Dhornetq.remoting.netty.host=localhost -Dhornetq.remoting.netty.port=5445"
 
-export JVM_ARGS="$CLUSTER_PROPS -XX:+UseParallelGC -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -Xms512M -Xmx1024M -Dhornetq.config.dir=$RESOLVED_CONFIG_DIR -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dlogging.configuration=file://$RESOLVED_CONFIG_DIR/logging.properties -Djava.library.path=./lib/linux-i686:./lib/linux-x86_64"
+export JVM_ARGS="$CLUSTER_PROPS -XX:+UseParallelGC -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -Xms512M -Xmx1024M -Dhornetq.config.dir=$RESOLVED_CONFIG_DIR -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dlogging.configuration=file://$RESOLVED_CONFIG_DIR/logging.properties -Djava.library.path=/usr/lib/hornetq/linux-i686:/usr/lib/hornetq/linux-x86_64"
 #export JVM_ARGS="-Xmx512M -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dlogging.configuration=$CONFIG_DIR/logging.properties -Dhornetq.config.dir=$CONFIG_DIR -Djava.library.path=. -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=5005"
 
-for i in `ls $HORNETQ_HOME/lib/*.jar`; do
+for i in `ls $HORNETQ_LIB/lib/*.jar`; do
 	CLASSPATH=$i:$CLASSPATH
 done
 
 echo "***********************************************************************************"
 echo "java $JVM_ARGS -classpath $CLASSPATH org.hornetq.integration.bootstrap.HornetQBootstrapServer $FILENAME"
 echo "***********************************************************************************"
-java $JVM_ARGS -classpath $CLASSPATH -Dcom.sun.management.jmxremote org.hornetq.integration.bootstrap.HornetQBootstrapServer $FILENAME
+java $JVM_ARGS -classpath $CLASSPATH -Dcom.sun.management.jmxremote org.hornetq.integration.bootstrap.HornetQBootstrapServer $FILENAME > /var/log/hornetq/hornetq_start-`date +%F_%H_%M_%S`.log 2>&1 &
--- a/bin/stop.sh
+++ b/bin/stop.sh
@@ -1,5 +1,6 @@
 #!/bin/sh
 
-export HORNETQ_HOME=..
-if [ a"$1" = a ]; then CONFIG_DIR=$HORNETQ_HOME/config/stand-alone/non-clustered; else CONFIG_DIR="$1"; fi
-touch $CONFIG_DIR/STOP_ME;
\ No newline at end of file
+. /etc/default/hornetq
+
+if [ a"$1" = a ]; then CONFIG_DIR=$HORNETQ_HOME/config/$HORNETQ_TYPE; else CONFIG_DIR="$1"; fi
+touch $CONFIG_DIR/STOP_ME;
--- a/config/stand-alone/clustered/logging.properties
+++ b/config/stand-alone/clustered/logging.properties
@@ -47,7 +47,7 @@
 handler.FILE.level=DEBUG
 handler.FILE.properties=autoFlush,fileName
 handler.FILE.autoFlush=true
-handler.FILE.fileName=logs/hornetq.log
+handler.FILE.fileName=${logs.dir}/hornetq.log
 handler.FILE.formatter=PATTERN
 
 # Formatter pattern configuration
--- a/config/stand-alone/non-clustered/logging.properties
+++ b/config/stand-alone/non-clustered/logging.properties
@@ -47,7 +47,7 @@
 handler.FILE.level=FINE
 handler.FILE.properties=autoFlush,fileName
 handler.FILE.autoFlush=true
-handler.FILE.fileName=logs/hornetq.log
+handler.FILE.fileName=${logs.dir}/hornetq.log
 handler.FILE.formatter=PATTERN
 
 # Formatter pattern configuration
--- a/config/stand-alone/replicated/logging.properties
+++ b/config/stand-alone/replicated/logging.properties
@@ -47,7 +47,7 @@
 handler.FILE.level=DEBUG
 handler.FILE.properties=autoFlush,fileName
 handler.FILE.autoFlush=true
-handler.FILE.fileName=logs/hornetq.log
+handler.FILE.fileName=${logs.dir}/hornetq.log
 handler.FILE.formatter=PATTERN
 
 # Formatter pattern configuration
--- a/config/stand-alone/shared-store/logging.properties
+++ b/config/stand-alone/shared-store/logging.properties
@@ -47,7 +47,7 @@
 handler.FILE.level=DEBUG
 handler.FILE.properties=autoFlush,fileName
 handler.FILE.autoFlush=true
-handler.FILE.fileName=logs/hornetq.log
+handler.FILE.fileName=${logs.dir}/hornetq.log
 handler.FILE.formatter=PATTERN
 
 # Formatter pattern configuration
